app.js

```diff
@@ -1,47 +1,147 @@
+  targetElement[0].remove();
+}
+
+function setTime() {
+  let target = document.getElementsByClassName('target');
+  if (totalmiliseconds == 0 && life > 0) {
+    lifeReduction();
+    deleteTarget();
+    if (target[0]){
+      deleteTarget();
+    }
+    createTarget();
+    createTarget();
+    reset();
+    startTimer();
+  }  else if (totalmiliseconds == 0 && life == 0){
+    lifeReduction();
+  }
+  --totalmiliseconds;
+  miliseconds.innerHTML = pad(totalmiliseconds) - (secondsElement.innerHTML*100)
+  secondsElement.innerHTML = pad(parseInt(totalmiliseconds/100));
+}
+
+function startTimer() {
+  isTimerRunning = true;
+  startAndResetButtonElement.textContent = 'stop';
+  intervalId = setInterval(setTime, 10);
+  
+}
+
+function reset () {
+  clearInterval(intervalId)
+  if (scoreBoardElement.innerHTML == 0) {
+    totalmiliseconds = 300
+  } else {
+    totalmiliseconds = Math.round(300 * (0.95 / scoreBoardElement.innerHTML));
+  }
+  miliseconds.innerHTML = 0
+  secondsElement.innerHTML = 0;
+}
+
+function stopTimer() {
+  isTimerRunning = false;
+  reset();
+  startAndResetButtonElement.textContent = 'Start the game';
+}
+
+function onClickstartTheGame() {
   if (isTimerRunning === false) {
-    isTimerRunning = true;
-    startAndResetButtonElement.textContent = 'stop';
-    intervalId = setInterval(setTime, 1000);
-    let paragrath = document.createElement('p');
-    boardElement.append("", paragrath);
-    paragrath.classList.add('board-element');
+    createTarget();
+    createTarget();
+    startTimer();
   } else if (isTimerRunning === true) {
-    isTimerRunning = false;
-    clearInterval(intervalId)
-    totalseconds = 0;
-    secondsElement.innerHTML = 0;
-    minutesElement.innerHTML = 0;
-    hoursElement.innerHTML = 0;
-    startAndResetButtonElement.textContent = 'Start the game';
+    stopTimer();
+    deleteTarget();
+    deleteTarget()
+    life = 3;
+    lifeBoardElement.innerHTML = 3;
+  }
+}
+
+function onHoveringTarget() {
+  console.log('hovering !');  
+}
+
+function onClickTarget() {
+  console.log('great success');
+  let target = document.getElementsByClassName('target');
+  deleteTarget();
+  if (!target[0]) {
+    score = score + 1;
+    scoreBoardElement.innerHTML = score;
+    createTarget();
+    createTarget();
+    reduction();
+    reset();
+    startTimer();
   }
 }
 
-startAndResetButtonElement.addEventListener('click', startTheGame);
\ No newline at end of file
+function lifeReduction() {
+  if (life > 0){
+    life = life - 1;
+    lifeBoardElement.innerHTML = life;
+  } else if (life == 0) {
+    stopTimer();
+    deleteTarget();
+    deleteTarget();
+    alert('Game over');
+    life = 3;
+    lifeBoardElement.innerHTML = 3;
+    width = 50;
+    height = 50;
+  }
+}
+
+function reduction() {
+  let target = document.getElementsByClassName('target');
+  width = Math.round(width * 0.95);
+  height = Math.round(height * 0.95);
+  target[0].style.width = width + 'px';
```

Comment:
Again you reduce size in the first element of the target array ! (the `[0]` index)

and you need to reduce size of every target but not currently displayed so the next generated target should have new size.